docs/manual/advanced-association-concepts/polymorphic-associations.md

```diff
@@ -424,4 +424,4 @@ WHERE (
 We can see that both scopes were applied automatically:
 
 * `` `tag_taggable`.`taggableType` = 'image'`` was added automatically to the `INNER JOIN`;
-* `` `tag`.`status` = 'pending'`` was added automatically to an outer where clause.
\ No newline at end of file
+* `` `tag`.`status` = 'pending'`` was added automatically to an outer where clause.

```

Comment:
what is the difference here? it looks like its just a git bug, they look identical to me.

Reply:
Yep, I didn't change anything, no change at all.